### PR TITLE
Fix #1230 - Untranslated strings

### LINKED ISSUE #1230: Untranslateable strings
"Data submitted!" when pressing the submit button that you can add in an app.
"From Scratch", "Teacher", "Business", "How to" titles when creating apps.

In Discover some apps says they are made by "guest " in english and then others says guest in my language "gæst". So mayne it should always be the same, so it always says "gæst" in my language even though its made by someone using a different language.

Go to Make and select From scratch. Select the menu in the top left corner.
"Back to My Apps", "Share", "Delete".

### PR DESCRIPTION
**Pull request by alicoding**
Closes #1230

### CODE REVIEW

**Comment by alicoding** [comment]
@k88hudson R?

File: locale/en_US/mobile-appmaker.json [hunk 1]
```diff
@@ -47,6 +47,10 @@
     "message": "My Apps",
     "description": "Mobile apps I have created"
   },
+  "My App": {
+    "message": "My App",
+    "description": "Mobile apps I am viewing"
```
Comment (k88hudson) [inline locale/en_US/mobile-appmaker.json]: The mobile app i am viewing

File: locale/en_US/mobile-appmaker.json [hunk 2]
```diff
@@ -279,6 +283,10 @@
     "message": "Phone #",
     "description": "Label for Phone # attribute on call/sms block"
   },
+  "App Data": {
+    "message": "App Data",
+    "description": "App details for currently viewing app"
```
Comment (k88hudson) [inline locale/en_US/mobile-appmaker.json]: Form data submitted for my app

**Comment by k88hudson** [comment]
@alicoding two small nits in descriptions, thanks for catching this!

**Pull request merged**